### PR TITLE
fabric: better test

### PR DESCRIPTION
**Pull request by ploxiln**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/fabric.rb [hunk 1]
```diff
@@ -12,8 +12,6 @@ class Fabric < Formula
     sha256 "bb39fa519461b1eb648b071a8cd06e13f4c10cda1fcd01b02a44be422019c37a" => :yosemite
   end
 
-  depends_on :python if MacOS.version <= :snow_leopard
```
Comment (nijikon) [inline Formula/fabric.rb]: @ilovezfs are we ok with this?

Comment (nijikon) [inline Formula/fabric.rb]: /cc @tdsmith

Comment (ploxiln) [inline Formula/fabric.rb]: I see in https://github.com/Homebrew/brew/blob/master/docs/Installation.md it says
> 10.10 or higher is recommended. 10.5 - 10.9 are supported on a best-effort basis.

So I'll get rid of this commit.

**Comment by nijikon** [comment]
You are awesome, thanks for your contribution! :tada:

**Pull request closed without merging**